ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: release
     namespace: openshift
     tag: rhel-9-release-golang-1.19-openshift-4.13
+  operator-sdk:
+    name: "4.14"
```

Comment:
Is this something we will have to update for each release?

Nit: I don't think the quotes are needed.

Reply:
I'm not sure. We have [other references](https://github.com/openshift/release/blob/c5ae5e624cfee501ab9ad8deabc76321cb7bcdfd/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml#L31) to the current OCP version which get auto updated. This may or may not be included in that auto-update tooling.

If not, us being slow to upgrade it is not the end of the world. It only means that we will use a slightly older operator-sdk version to deploy the operator.